### PR TITLE
polish api_cn for 1.6

### PR DESCRIPTION
**Pull request by tink2123**
modified:
1.XavierInitializer
2.acos
3.asin
4.atan
5.multiclass_nms
6.piecewise_decay
7.polynomial_decay
8.yolo_box
9.yolov3_loss
10.noam_decay

### CODE REVIEW

**Comment by tink2123** [comment]
预览效果：
1.XavierInitialize
image
2.acos
image
3.asin
image
4.atan
image
5. multicalss_nms
image
image
6.piecewise_decay
image
7.polynomial_decay
image
8.yolo_box
image
image
9.yolov3_loss
image
image
image

File: doc/fluid/api_cn/layers_cn/multiclass_nms_cn.rst [hunk 1]
```diff
@@ -7,32 +7,51 @@ multiclass_nms
 
 **多分类NMS**
 
-该运算用于对边界框（bounding box）和评分进行多类非极大值抑制（NMS）。
+该算子用于对边界框（bounding box）和评分（scores）执行多类非极大值抑制（NMS）。
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/multiclass_nms_cn.rst]: 该OP

File: doc/fluid/api_cn/layers_cn/multiclass_nms_cn.rst [hunk 2]
```diff
@@ -7,32 +7,51 @@ multiclass_nms
...
+                         [2, 0.4, 2.0, 3.0, 7.0, 5.0]]
+                         
+            输出数据格式为 (label, confidence, xmin, ymin, xmax, ymax)
 
-在NMS步骤后，如果keep_top_k大于-1，则每个图像最多保留keep_top_k个总bbox数。
 
 
 参数：
-    - **bboxes**  (Variable) – 支持两种类型的bbox（bounding box）:
+    - **bboxes**  (Variable) – 支持两种类型的边界框（bounding box）:
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/multiclass_nms_cn.rst:37]: 数据类型写一下

File: doc/fluid/api_cn/layers_cn/polynomial_decay_cn.rst [hunk 3]
```diff
@@ -17,11 +17,11 @@ polynomial_decay
             (1 - global_step / decay_steps) ^ power + end_learning_rate
 
 参数：
-    - **learning_rate** (Variable|float32)-标量float32值或变量。是训练过程中的初始学习率。
-    - **decay_steps** (int32)-Python int32数
-    - **end_learning_rate** (float)-Python float数
-    - **power** (float)-Python float数
-    - **cycle** (bool)-若设为true，每decay_steps衰减学习率
+    - **learning_rate** (Variable|float32) - 训练过程中的初始学习率，数据类型为float32的常数或变量。
+    - **decay_steps** (int32) - 衰减步数，数据类型为 int32
```
Comment (heavengate) [inline doc/fluid/api_cn/layers_cn/polynomial_decay_cn.rst]: decay_steps还有下面这些变量是python int型吧，应该不支持Variable？

Comment (tink2123) [reply doc/fluid/api_cn/layers_cn/polynomial_decay_cn.rst]: the above content has been fixed～ thx

File: doc/fluid/api_cn/initializer_cn/XavierInitializer_cn.rst [hunk 4]
```diff
@@ -22,9 +22,9 @@ XavierInitializer
     x = \sqrt{\frac{2.0}{fan\_in+fan\_out}}
 
 参数：
-    - **uniform** (bool) - 是否用均匀分布或者正态分布
-    - **fan_in** (float) - 用于Xavier初始化的fan_in。如果为None，fan_in沿伸自变量
-    - **fan_out** (float) - 用于Xavier初始化的fan_out。如果为None，fan_out沿伸自变量
+    - **uniform** (bool) - 是否用均匀分布，默认为True。如果为False，则使用正态分布。
+    - **fan_in** (float) - 当前网络层的输入神经元个数。如果为None，则从变量中推断。
+    - **fan_out** (float) - 当前网络层的输出神经元个数。如果为None，则从变量中推断。
```
Comment (ceci3) [inline doc/fluid/api_cn/initializer_cn/XavierInitializer_cn.rst]: 上面两个参数是不是需要写出默认值

File: doc/fluid/api_cn/layers_cn/acos_cn.rst [hunk 5]
```diff
@@ -11,16 +11,20 @@ arccosine激活函数。
     out = cos^{-1}(x)
 
 参数:
-    - **x** - acos算子的输入
+    - **x(Variable)** - acos的输入，数据类型为 float32 或 float64
+    - **name(None|str)** – 该参数供开发人员打印调试信息时使用，具体用法请参见 :ref:`api_guide_Name` ，默认值为None
 
-返回：        acos算子的输出。
+返回：  `acos` 的输出，数据类型与 `x` 相同。
```
Comment (ceci3) [inline doc/fluid/api_cn/layers_cn/acos_cn.rst]: 返回的时候需要写明是Tensor还是LoDTensor，同时需要写明返回类型？下同

File: doc/fluid/api_cn/layers_cn/noam_decay_cn.rst [hunk 6]
```diff
@@ -23,8 +25,8 @@ Noam衰减方法。noam衰减的numpy实现如下。
 请参照 `attention is all you need <https://arxiv.org/pdf/1706.03762.pdf>`_
 
 参数：
-    - **d_model** (Variable)-模型的输入和输出维度
-    - **warmup_steps** (Variable)-超参数
+    - **d_model** (Variable|int) - 模型的输入、输出向量特征维度。类型可设置为标量Tensor，或python int。
+    - **warmup_steps** (Variable|int) - 预热步数，类型可设置为标量Tensor，或python int。
```
Comment (ceci3) [inline doc/fluid/api_cn/layers_cn/noam_decay_cn.rst]: 需要直接写int？

**Review by ceci3 — APPROVED** [review]
LGTM

**Review by heavengate — APPROVED** [review]

**Pull request merged**